lib/travis/build/script/c.rb

```diff
@@ -3,7 +3,7 @@ module Build
     class Script
       class C < Script
         DEFAULTS = {
-          compiler: 'gcc'
+          compiler: ''
```

Comment:
Actually, you probably want to avoid setting `compiler` at all, instead of setting it to `''`?